src/extension/openFileAtLocation.ts

```diff
@@ -60,7 +60,8 @@ function getReactNativeWorkspaceForFile(file: string, workspace: string): Q.Prom
     }
     return getPathForRNParentWorkspace(path.dirname(file))
         .catch((reason) => {
-            return Q.reject<string>(ErrorHelper.getNestedError(reason, InternalErrorCode.WorkspaceNotFound, `Error while looking at workspace for file: ${file}.`));
+            const extraArgs = [{argument: `Error while looking at workspace for file: ${file}.`, isPii: true}];
```

Comment:
Do we really need to pass error message as argument?

Reply:
I added error with this code to the `errorStrings.ts` so  would be processed now as part of the WorkspaceNotFound error string